doc/source/cookbook.rst

```diff
@@ -1318,3 +1318,112 @@ of the data values:
        'weight': [100, 140, 180],
        'sex': ['Male', 'Female']})
    df
+
+Categorical Data
+----------------
+
+`Using categorical data type to store data more efficiently and consistently in multiple DataFrames
+<https://stackoverflow.com/questions/29709918/pandas-and-category-replacement/29712287#29712287>`
+
+`More information about categorical data <http://pandas.pydata.org/pandas-docs/stable/categorical.html>`
+
+.. ipython:: python
+
+   import pandas as pd
```

Comment:
I don't believe we include the standard import (pandas and numpy) anywhere else, they are assumed